community/dwm/build

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+
+# Amend install path to '/usr'
+sed -i'' "s#/usr/local#/usr#" config.mk
+
+# Remove pixel gap
+sed -i'' "s/static const int resizehints = 1/static const int resizehints = 0/" config.def.h
```

Comment:
Any specific reason for this change?

Reply:
Without the patch there is a weird 1pixel gap at the border of some windows. With the patch all windows just fill the screen as they should.

Reply:
:+1: